app/Http/Controllers/EventController.php

```diff
@@ -10,4 +10,12 @@ public function list()
     {
         return view('event_list');
     }
+<<<<<<< Updated upstream
```

Comment:
Please remove this, this will cause errors

Reply:
The code was fixed.